pk/hpke/hpke_test.c

```diff
@@ -0,0 +1,163 @@
+/* hpke_test.c
```

Comment:
This appears to just be a copy of the `hpke_test` in wolfCrypt test. Does it really make sense to duplicate the same code? Seems like a maintenance hassle, unless this provides something more?

Reply:
yes it's just a copy, If we can just point people to the test if they want an example I can remove this, I thought the point of this repo was to give easy to read standalone examples